### ISSUE TITLE
Add support for bundle config

### ISSUE THREAD

**Issue by ivanovv** [capistrano/bundler#35]
I know that capistrano/bundler supports env variables, but it would be great to support running [bundle config](http://bundler.io/v1.3/man/bundle-config.1.html) command before bundle install.
See [this question at StackOverflow](http://stackoverflow.com/questions/5167829/how-can-i-pass-a-parameter-for-gem-installation-when-i-run-bundle-install) as an example when it might be needed.

**Comment by thomasbiddle** [comment]
@kirs Solution is working fine for me; whether this is something that should be native or not is of course up for discussion though:

```ruby
task :prepare_bundle_config do
    on roles(:alibaba) do
      execute "bundle config mirror.https://rubygems.org https://ruby.taobao.org"
    end
  end
before 'bundler:install', 'deploy:prepare_bundle_config'
```

**Comment by mattbrictson** [comment]
This has now been implemented in #122 . Closing!
